src/session/session_prefetch.c

```diff
@@ -15,6 +15,10 @@
 bool
 __wt_session_prefetch_check(WT_SESSION_IMPL *session, WT_REF *ref)
 {
+    // Disable pre-fetch work on tiered tables.
```

Comment:
Could we change the comment to use `/* */` instead?

Reply:
Done!